customer/api.go

```diff
@@ -401,3 +401,36 @@ func (a *API) RequestEmailVerification(callbackURI string) error {
 		CallbackURI: callbackURI,
 	}, &emptyResponse{})
 }
+
+// GetDynamicConfiguration returns the dynamic configuration of a given group. It provides data to call Get Configuration and Get Localization.
+func (a *API) GetDynamicConfiguration(groupID int, url, channelType string, isTest bool) (*DynamicConfiguration, error) {
+	var resp DynamicConfiguration
+	err := a.Call("get_dynamic_configuration", &getDynamicConfigurationRequest{
```

Comment:
For all config endpoints:
1. `POST` -> `GET`
2. send params in query string, not json body